xbmc/video/VideoDatabase.cpp

```diff
@@ -348,22 +348,21 @@ void CVideoDatabase::CreateViews()
                                       "  tvshow.c%02d AS premiered,"
                                       "  tvshow.c%02d AS mpaa,"
                                       "  bookmark.timeInSeconds AS resumeTimeInSeconds, "
-                                      "  bookmark.totalTimeInSeconds AS totalTimeInSeconds, "
-                                      "  seasons.idSeason AS idSeason "
+                                      "  bookmark.totalTimeInSeconds AS totalTimeInSeconds "
                                       "FROM episode"
                                       "  JOIN files ON"
                                       "    files.idFile=episode.idFile"
                                       "  JOIN tvshow ON"
                                       "    tvshow.idShow=episode.idShow"
-                                      "  LEFT JOIN seasons ON"
-                                      "    seasons.idShow=episode.idShow AND seasons.season=episode.c%02d"
+                                      "  JOIN seasons ON"
```

Comment:
I'm pretty sure, I haven't found something wrong and I can't think of a case where am episode doesn't have a season. 
I know it's a bold change, but left joins are performance problems and need to go where possible.